install/installer/pkg/common/toleration.go

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2025 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License.AGPL.txt in the project root for license information.
+
+package common
+
+import (
+	configv1 "github.com/gitpod-io/gitpod/installer/pkg/config/v1"
+	corev1 "k8s.io/api/core/v1"
+)
+
+const (
+	RegistryFacadeTaintKey = "gitpod.io/registry-facade-not-ready"
+	WsDaemonTaintKey       = "gitpod.io/ws-daemon-not-ready"
```

Comment:
How will prereqs run on nodes with these taints? 

Do we need to add tolerations in prereqs now, too?

Reply:
Marking as resolved, following up here: https://github.com/gitpod-io/gitpod/pull/20652#issuecomment-2712629756